### DISCUSSION TITLE
Wall Time questions

### DISCUSSION THREAD

**Discussion by wilhelma** [DataDog/datadog-for-intellij-platform#5] [Q&A] [answered]
Dear Datadog team,

first of all, your plugin is super helpful for our developers to be aware of performance-critical changes!

We have three questions regarding the `Wall Time` metric, which is most useful for our APM approach.
1. How is the wall time metric calculated with a sampling profiler (we use the async-profiler option)? The question arises as the wall time metric is not shown in the Datadog `APM` > `Profiles view`. Sidenote: Why is this metric not shown here?
2. Why are the metric values in the flame graph view different from the values in the editor annotations (the latter are way lower than the ones in the flame graph view). This issue exists also with other metrics.
3. In our test, the wall time metric can not be used for short profile aggregation durations (e.g. 30 minutes). The `CPU time` metric shows results.

**Comment by vidurkhannadatadog** [comment, marked as answer]
Thank you for reaching out @wilhelma. We are glad that you find the plugin useful :) 

Answers to your questions:
1. Wall time profile is computed by sampling all threads. This is not shown in the Datadog web interface because it is in private beta right now.
2. We have identified the issue and are actively working on it. This will be fixed in the next release.
3. Can you please share more details on what you were trying to achieve and the issues you encountered?

**Reply by wilhelma** [reply]
Thank you for your answer and the fix for 2.

Regarding question 3, I was not able to get profiling data for the `Wall time` metric.
Data for the `CPU time` metric was available, though.
